apps/wallet-dashboard/components/Dialogs/Staking/timelock/UnstakeTimelockedObjectsDialog.tsx

```diff
@@ -0,0 +1,252 @@
+                        disabled={
+                            !timelockedUnstakeTx || isTransactionPending || isUnstakeTxLoading
+                        }
+                        type={ButtonType.Secondary}
+                        fullWidth
+                    />
+                </LayoutFooter>
+            </Layout>
+        </Dialog>
+    );
+}
+
+interface TimelockStakeObjectsOverviewProps {
+    groupedTimelockedObjects: TimelockedStakedObjectsGrouped;
+    handleCopySuccess: () => void;
+    rewardsPool: string | undefined;
+}
+
+function TimelockStakeObjectsOverview({
+    groupedTimelockedObjects: { stakes, validatorAddress, stakeRequestEpoch },
+    handleCopySuccess,
+    rewardsPool,
+}: TimelockStakeObjectsOverviewProps) {
+    const stakeId = stakes[0].timelockedStakedIotaId;
+
+    return (
+        <>
+            {stakeId && (
+                <ValidatorStakingData
+                    key={stakeId}
+                    validatorAddress={validatorAddress}
+                    stakeId={stakeId}
+                    isUnstake
+                />
+            )}
+
+            <Panel hasBorder>
+                <div className="flex flex-col gap-y-sm p-md">
+                    <KeyValueInfo
+                        keyText="Stake Request Epoch"
+                        value={stakeRequestEpoch}
+                        fullwidth
+                    />
+                    {rewardsPool && (
+                        <KeyValueInfo keyText="Rewards" value={rewardsPool} fullwidth />
+                    )}
+                    <KeyValueInfo keyText="Total Stakes" value={stakes.length} fullwidth />
+                </div>
+            </Panel>
+
+            {stakes.map((stake, index) => (
+                <Collapsible
+                    initialClose
+                    key={stake.timelockedStakedIotaId}
+                    title={'Stake' + ' ' + (index + 1)}
+                >
+                    <Panel>
+                        <div className="flex flex-col gap-y-sm p-md--rs py-sm">
+                            <KeyValueInfo
+                                keyText="Stake ID"
+                                value={formatAddress(stake.timelockedStakedIotaId)}
+                                hoverTitle={stake.timelockedStakedIotaId}
+                                onCopySuccess={handleCopySuccess}
+                                copyText={stake.timelockedStakedIotaId}
+                                fullwidth
+                            />
+                            <KeyValueInfo
+                                keyText="Expiration time"
+                                value={stake.expirationTimestampMs}
+                                fullwidth
+                            />
+                            {stake.label && (
+                                <KeyValueInfo
+                                    keyText="Label"
+                                    value={formatAddress(stake.label)}
```

Comment:
For the type, we could format it as the types are formatted in the explorer, with `formatType` function
example in TypeCard function `apps/explorer/src/pages/object-result/views/ObjectView.tsx`

Reply:
thats correct, my bad! 🙏🏼